### PR TITLE
Added client SSE plugin, updated test template

### PR DESCRIPTION
**Pull request by bjhham**
- Added a client entry for SSE
- Updated the server test template

Also, here's a sample PR for illustrating the file structure in the new plugin registry repository.

### CODE REVIEW

**Review by marychatte — APPROVED** [review]
Good work!

File: plugins/client/io.ktor/client-sse/3.0.0-beta-1/documentation.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Client "Server-sent Events" (SSE) plugin for consuming events sent using the
```
Comment (vnikolova) [inline plugins/client/io.ktor/client-sse/3.0.0-beta-1/documentation.md:1]: Client "Server-sent Events" (SSE) -> Server-Sent Events (SSE) plugin for the Ktor client.

File: plugins/client/io.ktor/client-sse/3.0.0-beta-1/documentation.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Client "Server-sent Events" (SSE) plugin for consuming events sent using the 
+[SSE specification](https://html.spec.whatwg.org/multipage/server-sent-events.html).
```
Comment (vnikolova) [inline plugins/client/io.ktor/client-sse/3.0.0-beta-1/documentation.md:2]: You could also refer to the docs: https://ktor.io/docs/3.0.0-beta-1/sse-client.html

File: plugins/server/io.ktor/sse/3.0.0-beta-1/documentation.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-Server-Sent Events (SSE) support plugin. It is required to be installed first before binding any sse endpoints.
+Server-Sent Events (SSE) support plugin for sending events using the [SSE specification](https://html.spec.whatwg.org/multipage/server-sent-events.html).
```
Comment (vnikolova) [inline plugins/server/io.ktor/sse/3.0.0-beta-1/documentation.md:1]: Docs reference: https://ktor.io/docs/3.0.0-beta-1/sse-server.html

**Review by vnikolova — CHANGES_REQUESTED** [review]
Not exactly related to this PR, but I notice the folders within the`plugins/client/io.ktor` folder have the `client-` prefix and I'd expect the ones  in `plugins/server/io.ktor` to have a `server-` prefix but they don't. Is that on purpose?

**Comment by bjhham** [comment]
I believe the `client-` prefix in the client plugins is because they came later and they need to be unique.  That's what I inferred from my code archeology anyway 😄 

I don't actually know where the client plugins are referenced from tbh 🤷

**Review by vnikolova — APPROVED** [review]

**Pull request merged**